src/background/adblocker.js

```diff
@@ -11,28 +11,22 @@
 
 import { FiltersEngine } from '@cliqz/adblocker';
 import { parse } from 'tldts-experimental';
+import { store } from 'hybrids';
+import Options, { DNR_IDS } from '/store/options.js';
 
-const adblockerEngines = {
-  'ads': {
+const adblockerEngines = DNR_IDS.reduce((map, name) => {
+  map[name] = {
     engine: null,
     isEnabled: false,
-  },
-  'tracking': {
-    engine: null,
-    isEnabled: false,
-  },
-  'annoyances': {
-    engine: null,
-    isEnabled: false,
-  },
-};
+  };
+  return map;
+}, {});
 
 export async function updateAdblockerEngineStatuses() {
-  const enabledRulesetIds =
-    await chrome.declarativeNetRequest.getEnabledRulesets();
-  Object.keys(adblockerEngines).map((engineName) => {
-    adblockerEngines[engineName].isEnabled =
-      enabledRulesetIds.indexOf(engineName) > -1;
+  const options = await store.resolve(store.get(Options));
```

Comment:
would be nice to have an alias `await store.untilReady(Options)`

Reply:
No for aliases! :) No choice is always better than having a pointless choice. It does what it says ;)